src/components/pages/CategoryDetailPage/CategoryDetailPage.tsx

```diff
@@ -53,42 +47,30 @@ function CategoryDetailPage() {
     fetchNextPage,
     isFetching,
     refetch,
-  } = useInfiniteQuery(
-    ["categoryStore", { categoryId, campusId }],
-    fetchCategoryStoreList,
-    {
-      getNextPageParam: (lastPage) => {
-        if (lastPage.hasNext) {
-          return lastPage.nextPageParam;
-        }
-        return;
-      },
-    }
-  );
+  } = useInfiniteQuery(["categoryStore", fetchParams], fetchStoreList, {
+    getNextPageParam,
+  });
 
-  // isSelected 선택 된 값이 달라질때마다 refetch 해서 데이터 업데이트
   useEffect(() => {
     refetch();
-  }, [isSelected]);
+  }, [filter]);
 
   const loadMoreStores = () => {
     fetchNextPage();
   };
 
-  // TODO: 클릭한 값에 따라 stores prop 바꾸기
   const handleClickStarOrderChip = () => {
-    setIsSelected((prev) => ({
-      starOrder: !prev.starOrder,
-      abcOrder: false,
-    }));
+    setFilter("rating");
```

Comment:
이거 이렇게 하면 둘 다 끄는 건 못하네!
```suggestion
    setFilter(prev => prev ? "" : "rating");
```
이렇게 해야 둘 다 끌 수 있을 듯!